static/app/components/dropdownMenuControlV2.tsx

```diff
@@ -127,8 +127,10 @@ function MenuControl({
   const [triggerWidth, setTriggerWidth] = useState<number>();
   // Update triggerWidth when its size changes using useResizeObserver
   const updateTriggerWidth = useCallback(() => {
-    const newTriggerWidth = ref.current?.offsetWidth;
-    !isSubmenu && newTriggerWidth && setTriggerWidth(newTriggerWidth);
+    setTimeout(() => {
```

Comment:
@vuluongj20 I'm wondering if `requestAnimationFrame()` is a modern alternative as suggested in https://github.com/adobe/react-spectrum/issues/1924#issuecomment-871623619